pagopa/io-app
# chore: [IOBP-529] Retrieve payments methods list from eCommerce client endpoint

## Short description
This PR replaces wallet client endpoints in the payment flow with the eCommerce client endpoints

## List of changes proposed in this pull request
- Updated eCommerce API definitions
- Replaced API calls in payment flow sagas

## How to test
In **UAT environment**, start a new payment flow from **Profile > Playgrounds > New Wallet > Payment** using the following data:
- Payment number: **302000190000909881** (add 1 if not working)
- Fiscal code: **77777777777**

You should be able to finalize a payment with a success outcome.
